pwiz_tools/Skyline/SettingsUI/PeptideGridViewDriver.cs

```diff
@@ -17,9 +17,11 @@
  * limitations under the License.
  */
 using System;
+using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
 using System.Windows.Forms;
+using NHibernate.Transform;

```

Comment:
Seems like this might be the cause of the code inspection failure? Can't really see how NHibernate would be required for your changes.